aidial_adapter_bedrock/llm/model/stability/v2.py

```diff
@@ -0,0 +1,205 @@
+        self,
+        consumer: Consumer,
+        params: ModelParameters,
+        messages: List[Message],
+    ) -> None:
+
+        text_prompt = None
+        image_resources: List[DialResource] = []
+        last_message = self._validate_last_message(messages)
+        # Handle text content
+        match last_message.content:
+            case str(text):
+                text_prompt = text
+            case list():
+                text_parts = []
+
+                for part in last_message.content:
+                    match part:
+                        case MessageContentTextPart(text=text):
+                            text_parts.append(text)
+                        case MessageContentImagePart(
+                            image_url=ImageURL(url=url)
+                        ):
+                            image_resources.append(
+                                URLResource(
+                                    url=url,
+                                    supported_types=SUPPORTED_IMAGE_TYPES,
+                                )
+                            )
+                        case _:
+                            assert_never(part)
+                if text_parts:
+                    text_prompt = " ".join(text_parts)
+            case None:
+                pass
+            case _:
+                assert_never(last_message.content)
+
+        if (
+            last_message.custom_content
+            and last_message.custom_content.attachments
+        ):
+            image_resources.extend(
+                [
+                    AttachmentResource(
+                        attachment=attachment,
+                        supported_types=SUPPORTED_IMAGE_TYPES,
+                    )
+                    for attachment in last_message.custom_content.attachments
+                ]
+            )
+
+        if not self.image_to_image_supported and image_resources:
+            raise ValidationError(
+                f"Image-to-image is not supported for {self.model}"
+            )
+        if len(image_resources) > 1:
+            raise ValidationError("Only one input image is supported")
+        try:
+            response, _ = await self.client.ainvoke_non_streaming(
+                self.model,
+                remove_nones(
+                    {
+                        "prompt": text_prompt,
+                        "image": (
+                            (
+                                await image_resources[0].download(self.storage)
+                            ).data_base64
+                            if image_resources
+                            else None
+                        ),
+                        "mode": (
+                            "image-to-image"
+                            if image_resources
+                            else "text-to-image"
+                        ),
+                        "output_format": "png",
+                        "strength": 0.5 if image_resources else None,
```

Comment:
There is no default value for this required parameter, non in Bedrock, non in Stability Docs

Basically this parameter means, how much input image will affect generation, where `0 - output will be identical to input image`, `1 - model will ignore input image`


I think at some point this parameter can be moved to `.env`, but I didn't want to do this now, since there is only 1 model for now with this feature, and we don't know anything about demand for such parametrization

Reply:
Could you add this comment to the code? Something like this:

```
Settings a reasonable default for the required parameter that controls how much input image will affect generation: 0 - output will be identical to input image, 1 - model will ignore input image
```

Reply:
Done